src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3328,6 +3348,8 @@ def construct_mc_profile_preview(self, bypass_restore_defaults: bool = False) ->
         mc = self.set_up_node_provisioning_profile(mc)
         # set up agentpool profile ssh access
         mc = self.set_up_agentpool_profile_ssh_access(mc)
+        # set up network profile
+        mc = self.set_up_network_profile(mc)
```

Comment:
may remove this, it's already called in `mc = self.construct_mc_profile_default(bypass_restore_defaults=True)`

Reply:
removed